syspurpose/src/syspurpose/files.py

```diff
@@ -553,14 +572,17 @@ def three_way_merge(local, base, remote, on_conflict="remote", on_change=None):
         source = 'base'
 
         if local_changed == remote_changed:
+            log.debug('Three way merge conflict: both local and remote values changed for key \'%s\'.' % key)
```

Comment:
If we want to keep this message as phrased, I'd suggest surrounding it in:
```python
if local_changed == True:
    log.debug('Three way merge conflict: both local and remote values changed for key \'%s\'.' % key)
```

The reason is for the case where both values are false, in that case, neither of them has changed the value.